front-end/src/components/player-page/game-summary/game-time-information.tsx

```diff
@@ -1,24 +1,34 @@
 import {Box, Typography} from "@mui/material";
 import React from "react";
-import {riotTimestampToGameTime} from "../../../../../Common/utils";
+import {riotTimestampToGameTime, timeToTimeAgo} from "../../../../../Common/utils";
 
 interface GameTimeInformationProps {
     playerWin: boolean;
-    gameDuration: number
+    gameDuration: number;
+    gameStart: number;
 }
 
 function GameTimeInformation(gameTimeInformationProps: GameTimeInformationProps) {
-    return <Box sx={{display: 'flex', flexDirection: 'column', justifyContent: 'center'}}>
-        <Box sx={{width: "40px"}}>
+    const {
+        playerWin,
+        gameDuration,
+        gameStart,
+    } = gameTimeInformationProps;
```

Comment:
Is there a reason you cant just do gameTimeInformationProps.playerWin in the code below? This assignment is probably unneeded

Reply:
Beacuse I like how this looks. It keeps it shorter in the return value.